doc/whats_new.rst

```diff
@@ -21,6 +21,10 @@ Fixes
 -----
 
 - :class:`nilearn.input_data.NiftiLabelsMasker` no longer ignores its `mask_img`
+- :class:`nilearn.masking.compute_gray_matter_mask` has been renamed to
```

Comment:
```suggestion
- :func:`nilearn.masking.compute_gray_matter_mask` has been renamed to
```

Reply:
Thanks for the comments. I am solving them. Should I change also the name of the function `test_compute_gray_matter_mask`?: https://github.com/nilearn/nilearn/blob/02b6ae5b0a45a4ac3a8644360a652bb0515b337d/nilearn/input_data/tests/test_nifti_masker.py#L317